app/gui2/ydoc-server/languageServerSession.ts

```diff
@@ -70,13 +70,16 @@ export class LanguageServerSession extends ObservableV2<Events> {
       }
     })
 
-    this.ls.on('file/event', (event) => {
+    this.ls.on('file/event', async (event) => {
       if (DEBUG_LOG_SYNC) {
         console.log('file/event', event)
       }
       switch (event.kind) {
         case 'Added':
-          this.getModuleModel(event.path).open()
+          const fileInfo = await this.ls.fileInfo(event.path)
```

Comment:
Additionaly before asking for the file info, we can also early-out when a file path is not matching `src/**/*.enso` format.

Reply:
Yep, I found it necessary, as any non-text file would also fail. I will update the PR in a moment.